save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ProjectController.kt

```diff
@@ -123,10 +123,10 @@ class ProjectController(private val projectService: ProjectService,
      * @return project by name and organization name
      */
     @GetMapping("/get/projects-by-organization")
-    @PreAuthorize("hasRole('VIEWER')")
+    @PreAuthorize("permitAll()")
```

Comment:
But what with private projects after this?

Reply:
They are filtered out by `hasPermission` check

Reply:
Yes, but formally, we didn't need this `hasRole('VIEWER')` annotation before, since anyway we will correctly filter out only necessary once by `hasPermission`?

Reply:
Yes, but I think having those annotations on all methods makes code more consistent, especially since this controller is now public. It just shows that we didn't forget about permission check, it's just sort of no-op.